inspirehep/modules/forms/validators/simple_fields.py

```diff
@@ -42,16 +39,6 @@ def arxiv_syntax_validation(form, field):
         raise StopValidation(message)
 
 
-def isbn_syntax_validation(form, field):
```

Comment:
What about ISBNs? @jmartinm how do we validate them? Or it's simply because users can't suggest books?